connect/file/src/main/java/org/apache/kafka/connect/file/FileStreamSourceTask.java

```diff
@@ -63,12 +59,7 @@ public String version() {
     @Override
     public void start(Map<String, String> props) {
         filename = props.get(FileStreamSourceConnector.FILE_CONFIG);
```

Comment:
moved the logic from line 66-71 to FileStreamBuffer.setFilename and FileStreamBuffer.ensureOpen